debian/decrypt-files.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/sh
+
+set -eu
+
+pg_version=
+libdir="/usr/lib/postgresql/$pg_version/lib"
+secret_files_list="$libdir/pgautofailover_secret_files.metadata"
+
+# Make sure the script is being run as root
+if [ "$(id -u)" -ne "0" ]; then
+    echo "ERROR: pg-auto-failover-enterprise-pg-$pg_version-setup needs to be run as root"
+    echo "HINT: try running \"sudo pg-auto-failover-enterprise-pg-$pg_version-setup\" instead"
+    exit 1
+fi
+
+
+echo "
+Your use of this software is subject to the terms and conditions of the license
+agreement by which you acquired this software. If you are a volume license
+customer, use of this software is subject to your volume license agreement.
+You may not use this software if you have not validly acquired a license for
+the software from Microsoft or its licensed distributors.
+
+BY USING THE SOFTWARE, YOU ACCEPT THIS AGREEMENT.
+"
+
+CITUS_ACCEPT_LICENSE="${CITUS_ACCEPT_LICENSE:-}"
+
+interactive_license=false
+while [ -z "$CITUS_ACCEPT_LICENSE" ]; do
+    interactive_license=true
+    echo "Do you accept these terms? YES/NO"
+    read -r CITUS_ACCEPT_LICENSE
+done
+
+case "$CITUS_ACCEPT_LICENSE" in
+    YES );;
+    y|Y|Yes|yes )
```

Comment:
This seems really unintuitive to me. I am used to typing a single letter (y or n) when such a prompt appears without actually reading all my options.

If there is a solid reason thy only `YES` is allowed, I'm totally ok with this

Reply:
Legel team prefered it that way. (It's the same for the citus enterprise microsoft packages.)